add rune update-info command (#206)
rune/rune: docs/docs/publishing/cli.md

Displays a list of your games on Rune.

### `rune update-info`

> Allows to update you game's info such as title, description or logo.

Reviewer: ```suggestion
Update your game's info such as title, description or logo.
```
